fetch_prod.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Fetches anonymized prod dump for development seeding
+set -ex
+
+user=$1
+
+cat ./infrastructure/prod/anonymize.sh | ssh -J $user@login.ccs.neu.edu $user@amato.ccs.neu.edu
```

Comment:
AFAIK bash is supposed to wait for the `ssh` session to be over before continuing w/ the script. instead of using `cat`, might be better to just move `./infrastrucutre....` to the RHS of `ssh`